src/wasm-interpreter.h

```diff
@@ -4300,11 +4607,105 @@ class ModuleRunnerBase : public ExpressionRunner<SubType> {
     multiValues.pop_back();
     return ret;
   }
-  Flow visitContNew(ContNew* curr) { return Flow(NONCONSTANT_FLOW); }
+  Flow visitContNew(ContNew* curr) {
+    auto funcFlow = self()->visit(curr->func);
+    if (funcFlow.breaking()) {
+      return funcFlow;
+    }
+    // Create a new continuation for the target function.
+    Name func = funcFlow.getSingleValue().getFunc();
+    return Literal(std::make_shared<ContData>(func, curr->type.getHeapType()));
+  }
   Flow visitContBind(ContBind* curr) { return Flow(NONCONSTANT_FLOW); }
+  Flow visitSuspend(Suspend* curr) {
+    if (self()->resuming) {
+      // This is a resume, so we have found our way back to where we
+      // suspended.
+      assert(curr == self()->currContinuation->resumeExpr);
+      // We finished resuming, and will continue from here normally.
+      self()->resuming = false;
+      // We should have consumed all the resumeInfo and all the
+      // restoredValues map.
+      assert(self()->currContinuation->resumeInfo.empty());
+      assert(self()->restoredValuesMap.empty());
+      return Flow();
```

Comment:
Suspend should be able to return values.

Reply:
I will have testing + a fix for this in the next PR. I tried to keep this one minimal - no support for calls/params/results/etc.

Adding those isn't hard, but I was hoping this PR, that focuses on pause/resume, would be easier to follow that way.